### ISSUE TITLE
PractitionerRole slice for NPIO - system fixed value queries

### ISSUE THREAD

**Issue by robeastwood-agency** [hl7au/au-fhir-base#157]
The slice on [PractitionerRole.identifier(nationalProviderAtOrganisation)](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-practitionerrole-definitions.html#PractitionerRole.identifier:nationalProviderAtOrganisation) has the [fixed value for system](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-practitionerrole-definitions.html#PractitionerRole.identifier:nationalProviderAtOrganisation.system) of this:
http://hl7.org.au/id/npio.

This resolves but the text description has "_Australian Health Practitioner Regulation Agency Registration Number_" and probably should be something like "_National Provider Identifer at Organisation (NPIO)_" or similar.

Also noting that the slice is fairly light on details (ie what is this for etc) and therefore suggest to have additional text added to the namespace page to better explain the use. I found on [a O&O confluence page](https://confluence.hl7australia.com/display/OO/6+Identifiers) this description: 
> An identifier that is unique for an individual healthcare provider at a healthcare organisation. It is the concatenation of a HPI-I identifier followed by an "@" and then a HPI-O identifier. e.g. "800361XXXXXXXXXX@800362YYYYYYYYYY"

**tl;dr** - looks like a placeholder namespace needing some attention

**Comment by oridashi** [comment]
thanks - fixed wording uploaded
